entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+
+# TODO: share code with bib to do the setup automatically
+# see https://github.com/teamsbc/container-for-osbuild/blob/main/entrypoint.bash (thanks simon)
+# and https://github.com/osbuild/bootc-image-builder/blob/main/bib/internal/setup/setup.go#L21 (thanks ondrej,achilleas,colin)
+mkdir /run/osbuild
+mkdir /run/osbuild-store
+
+mount -t tmpfs tmpfs /run/osbuild
+mount -t tmpfs tmpfs /run/osbuild-store
```

Comment:
I think this is currently unused, and I think we should actually use a "real" filesystem for the osbuild store, because tmpfs might not be big enough for image building.

Reply:
Thanks, indeed, I removed this and followed our pattern from bib and added `--store` and use that in the entrypoint.